fcrepo-persistence-api/src/main/java/org/fcrepo/persistence/api/exceptions/PersistentStorageException.java

```diff
@@ -15,12 +15,30 @@
  * See the License for the specific language governing permissions and
```

Comment:
let's remove this for now.

Reply:
Remove the license?

Reply:
Or was this from when you wanted to remove the entire exception?